src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -61,6 +61,21 @@ public static void Main(string[] args)
         }
 
         [Fact]
+        public void GetAssemblyFileVersionOnDevBuild()
+        {
+            var cmd = new MockCSharpCompiler(null, _baseDirectory, new[] { "/nologo", "/preferreduilang:en" });
+            Assert.Equal("42.42.42.42424 (<developer build>)", cmd.GetAssemblyFileVersion());
+
+            Assert.Equal("<developer build>", CommonCompiler.ExtractShortSHA("2.1.0-dev-56735-00. Commit Hash: <developer build>"));
+            Assert.Equal("12345678", CommonCompiler.ExtractShortSHA("2.1.0-dev-56735-00. Commit Hash: 12345678b6d4d10cf04e128efc6ed56f9e424f36"));
+            Assert.Equal("", CommonCompiler.ExtractShortSHA("2.1.0-dev-56735-00. commit hash: 12345678b6d4d10cf04e128efc6ed56f9e424f36")); // casing
+            Assert.Equal("", CommonCompiler.ExtractShortSHA("2.1.0-dev-56735-00. Commit Hash: 1bc"));
+            Assert.Equal("", CommonCompiler.ExtractShortSHA("xyz"));
+            Assert.Equal("", CommonCompiler.ExtractShortSHA(""));
+            Assert.Equal("", CommonCompiler.ExtractShortSHA(null));
```

Comment:
Thanks for the suggestions. The second one `ExtractShortSHA("2.0.0.0 Commit Hash: ")` was crashing :-S